src/test/ui/array-slice-vec/infer_array_len.stderr

```diff
@@ -4,7 +4,7 @@ error[E0282]: type annotations needed
 LL |     let [_, _] = a.into();
    |         ^^^^^^
    |
-help: consider giving this pattern a type
+help: consider giving this pattern a type, where the placeholders `_` are specified
```

Comment:
It's unfortunate that the message is kind of ambiguous here in the presence of those two other `_` inside of the slice pattern. By a beginner, it could be interpreted as if `[_, _]` needs to be adjusted, too. I don't have any suggestions though on how this could potentially be improved.

Reply:
I noticed that as well, but it is better than providing *no* mention of "replacing _ with a type". An alternative might be to instead of using _ as a placeholder I used `Type` as a placeholder for the case where the suggested type is `ty::Infer` and mentioned that by name.